storybook/src/elements/nuxeo-date/nuxeo-date.stories.js

```diff
@@ -0,0 +1,23 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date';
+
+storiesOf('UI/nuxeo-date', module)
+  .add(
```

Comment:
Cool! Thanks for your contribution.

I confess I'm not sure what is better... but I was just wondering if it would be more interesting to have an "interactive story" with knobs where you could select/set the properties (i.e. datetime, format, tooltipFormat). WDYT?
For instance, if we want to have some formats as example, we can rely on momentjs formats (see here https://momentjs.com/docs/)